maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -195,14 +194,40 @@ const LayerControlPanel = memo(({ maplibreRef, setLayers, layers }: Props) => {
     layerVisibility.set(layer.id, layer.visibility === LAYER_VISIBILITY.VISIBLE);
   });
 
+  const beforeMaplibreLayerID = (source: number, destination: number) => {
+    if (source > destination) {  // if layer is moved below, move current layer below given destination
+      return layers[destination].id;
+    }
+    const beforeIndex = destination + 1; // if layer is moved up, move current layer above destination
+    if (beforeIndex < layers.length) {
+      return layers[beforeIndex].id;
+    }
```

Comment:
do we cover all possible cases with those two "if" statements? maybe join them into if/elseif something like:

```
 if (source > destination) {  // if layer is moved below, move current layer below given destination
      return layers[destination].id;
    } else if (destination + 1 < layers.length) { // if layer is moved up, move current layer above destination
      return layers[destination + 1].id;
    }
```

Reply:
Yes. since there is return before else if, i avoided else